src/services/api/handlers/submit.ts

```diff
@@ -53,9 +56,9 @@ export const submit = async (event: APIGatewayEvent) => {
     }
 
     const activeSubmissionTypes = [
-      PlanType.CHIP_SPA,
-      PlanType.MED_SPA,
-      PlanType["1915b"],
+      Authority.CHIP_SPA,
+      Authority.MED_SPA,
+      Authority["1915b"],
```

Comment:
There's probably further work here to unwind _this_ use of the Authority idea. To me, it looks like we're introducing `Flavors` as the very same idea as this enum, and just renaming the enum for now rather than refactoring it out. Is that correct?